thrift/admin.thrift

```diff
@@ -252,9 +252,15 @@ struct MembershipInfo {
   30: optional list<RingInfo> rings
 }
 
+struct PersistenceInfo {
+  10: optional string name
```

Comment:
Sorry I guess I left the same comment in the server side pr. Is it possible to rename this field so that in the future we can add persistence info for the main storage without causing confusion? Looks like for right now this name field is just for visibility persistence.